### ISSUE TITLE
Is there a typo in 1.4.13?

### ISSUE THREAD

**Issue by goodwitch** [w3c/wcag#420]
I was just running some WCAG 2.1 content through a spell checker and got an error on the word "dismissable".  The spell checker I'm using suggested "dismissible" instead.  When I checked dictionary.com it also said "dismissable" was not a word.

So, I'm thinking we have a typo in https://www.w3.org/TR/WCAG21/#content-on-hover-or-focus

What do y'all think?

**Comment by eadraffan** [comment]
The Oxford English Dictionary only has 'dismissible' and I would have said that you are right there is a typo!     <https://en.oxforddictionaries.com/definition/dismissible> https://en.oxforddictionaries.com/definition/dismissible 

 

However, Google provided 2,760 results for 'dismisssable' so lots of people are making the same mistake! 

 

Best wishes 

E.A.

**Comment by awkawk** [comment]
Sounds like this should be another editorial errata. Any opposed?

**Comment by johnfoliot** [comment]
+1, no opposition.

JF

**Comment by Ryladog** [comment]
+1

** katie **

*Katie Haritos-Shea*

*Principal ICT Accessibility Architect, **Board Member and W3C Advisory
Committee Rep for Knowbility *

*WCAG/Section 508/ADA/AODA/QA/FinServ/FinTech/Privacy,* *IAAP CPACC+WAS = *
*CPWA* <http://www.accessibilityassociation.org/cpwacertificants>

*Cell: **703-371-5545 <703-371-5545>** |* *ryladog@gmail.com
<ryladog@gmail.com>* *| **Oakton, VA **|* *LinkedIn Profile
<http://www.linkedin.com/in/katieharitosshea/>*

People may forget exactly what it was that you said or did, but they will
never forget how you made them feel.......

Our scars remind us of where we have been........they do not have to
dictate where we are going.

**Comment by mbgower** [comment]
My "Shorter Oxford" (i.e., only 2 volumes) shows _dismissable_ first appearing between 1800-1829 with _dismissible_ coming into usage in the following 3-decade period. The latter is preferred but both are acceptable and mean the same thing.
If the w3c has no authority for word usage, I think we can decide on either, and the version with no "a" seems to be the more common spelling.

**Comment by awkawk** [comment]
fixed.
